internal/services/servicebus/servicebus_subscription_data_source.go

```diff
@@ -35,34 +36,6 @@ func dataSourceServiceBusSubscription() *pluginsdk.Resource {
 				Type:         pluginsdk.TypeString,
 				Optional:     true,
```

Comment:
This should be set to `Required` and updated to `Optional` in the `FivePointOh` block below